CentrED/Map/MapManager.cs

```diff
@@ -549,7 +549,7 @@ public void RemoveTiles(ushort x, ushort y)
         }
     }
 
-    public IEnumerable<TileObject> GetTopTiles(TileObject? t1, TileObject? t2)
+    public IEnumerable<TileObject> GetTopTiles(TileObject? t1, TileObject? t2, bool altMode = false)
```

Comment:
Now it does more than just `GetTopTiles` :D
As it's more than that I would rename it to just `GetTiles` and make that bool parameter to be `topTilesOnly`
We need to basically revert the logic :D
But this way it would be much more understandable as `altMode` says almost nothing what it does

Reply:
ok, but you want the default value of topTilesOnly true or false?

Reply:
We can totally remove the default value to make sure that we use the correct mode :D

Reply:
so we keep one key to affects only top tiles and two key to affects all tiles, right?

Reply:
Yes, this should stay unchagend, we just want a little bit more clarity in the code from my suggested changes

Reply:
ok done